Formula/d/dug.rb

```diff
@@ -0,0 +1,22 @@
+class Dug < Formula
+  desc "Global DNS propagation checker that gives pretty output"
+  homepage "https://dug.unfrl.com"
+  url "https://github.com/unfrl/dug/archive/refs/tags/0.0.93.tar.gz"
+  sha256 "a375034bcad3193d47a485b3dd33c7633f7890e5f79b6a9b6b3b3bc198fb2ec3"
+  license "MIT"
+
+  depends_on "dotnet"
+  uses_from_macos "zlib"
+
+  def install
+    ENV["DOTNET_SYSTEM_GLOBALIZATION_INVARIANT"] = "1"
+    system "dotnet", "publish", "-c", "Release", "-p:PublishSingleFile=true",
+            "-p:IncludeNativeLibrariesForSelfExtract=true", "--self-contained",
```

Comment:
should be --no-self-contained mode?

Reply:
ah, I see what you other comment means now. `write_env_script` sets an envvar for an installed bin. Gotcha.